disco_aws_automation/disco_rds.py

```diff
@@ -177,25 +179,27 @@ def setup_dns(self, instance_identifier):
         disco_route53.delete_record(self.domain_name, instance_record_name, 'CNAME')
         disco_route53.create_record(self.domain_name, instance_record_name, 'CNAME', instance_endpoint)
 
-    def spinup_alarms(self, database_class):
+    def spinup_alarms(self, database_name):
         """
         Configure alarms for this RDS instance. The alarms are configured in disco_alarms.ini
         """
         logging.debug("Configuring Cloudwatch alarms ")
         disco_alarm_config = DiscoAlarmsConfig(self.vpc_name)
         disco_alarm = DiscoAlarm()
-        instance_alarms = disco_alarm_config.get_alarms(database_class)
+        instance_alarms = disco_alarm_config.get_alarms(database_name)
         disco_alarm.create_alarms(instance_alarms)
 
     def update_all_clusters_in_vpc(self):
         """
         Updates every RDS instance in the current VPC to match the configuration
         """
         sections = [section for section in self.config_rds.sections()
-                    if section.split("-")[0] == self.vpc_name]
+                    if section.startswith(self.vpc_name) + '-']
```

Comment:
It makes me twitch to see the same string concatenation done in every iteration of a loop.  Not sure it actually matters, though.